packages/pds-ember/addon/components/pds/loading/elapsed/index.js

```diff
@@ -0,0 +1,78 @@
+import Component from '@glimmer/component';
+import { run } from '@ember/runloop';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+
+/**
+ * Calculate the time difference between now and given argument.
+ *
+ * @function
+ * @param {number} startTime - Unix Epoch in milliseconds
+ * @return {string} elapsed time in `[HH:]MM:SS` format
+ */
+// TODO: unit test or convert to date-fns (or Temporal)
+//       to handle calculation and formatting
+function timeSince(startTime) {
+  let now = new Date().getTime()
+  let diff = now - startTime
+  // TODO: consider using date-fns (or JS:Temporal) to calculate this
+  let hours = Math.floor((diff % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
+  let minutes = Math.floor((diff % (1000 * 60 * 60)) / (1000 * 60));
+  let seconds = Math.floor((diff % (1000 * 60)) / 1000);
+  // TODO: consider using date-fns (or JS:Temporal) to format the duration
+  hours = hours < 10 ? '0' + hours : hours;
```

Comment:
do you want to remove the semi-colons from here?

Reply:
I'll update. Thanks for pointing these out.